### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: Dockerfile-mayhem [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Build Stage:
```
Comment (xansec) [inline Dockerfile-mayhem:1]: Can we add a package stage here?

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Namespaced project name that the target belongs to
```
Comment (xansec) [inline Mayhemfile:1]: Please use generic project name, and omit the image.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by bmwtsn098** [comment]
I made the requested changes. Let me know if everything looks good. Thanks

**Pull request merged**